packages/engine-test-utils/src/__tests__/engine-server/markdown/backlinks.spec.ts

```diff
@@ -52,7 +52,7 @@ describe("backlinks", () => {
         expect(
           await AssertUtils.assertInString({
             body: resp.contents as string,
-            match: [`<a href="alpha.html">Alpha (vault1)</a>`],
+            match: [`<a href="alpha">Alpha (vault1)</a>`],
```

Comment:
the tests here are testing against the config v4 defaults (with enablePrettyLinks set to false). can these tests continue to test against the v4 defaults? we can create new tests for v5 defaults. no need to be exhaustive about the v5 tests by the v4 tests shouldn't change

also note: there is a v5 folder in the `markdown` folder which can be confusing. this refers to `MDUtilsV5` and is not related to the configuration.